best_practices/templates.rst

```diff
@@ -51,6 +51,10 @@ Another advantage is that centralizing your templates simplifies the work
 of your designers. They don't need to look for templates in lots of directories
 scattered through lots of bundles.
 
+.. best-practice::
+
+    Use lowercased snake_case for directory and template names.
```

Comment:
@javiereguiluz What do you think? We've never talked too much about this directly. The negative to underscoring everything is that templates aren't the same as their controllers anymore - PostAdminController::createPostAction would be in post_admin/create_post.html.twig.

We're just talking about a recommendation here - not life or death - but let's think about it. The old upper-camel case was done without really thinking, to support the `@Template` magic (because we were all very used to having this from 1.x). If we break from this, then `@Template` won't work, and even if we update it, trying to transform from `createPostAction` to `create_post.html.twig` could be buggy.

What do guys think?